windows_containers/wmco_rn/windows-containers-release-notes-10-17-x-prereqs.adoc

```diff
@@ -4,7 +4,14 @@
 = Windows Machine Config Operator prerequisites
 include::_attributes/common-attributes.adoc[]
 
-The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
+The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator (WMCO). See the vSphere documentation for any information that is relevant to only that platform.
+
+[id="wmco-prerequisites-supported-install-10.17.0_{context}"]
+== WMCO supported Installation method
```

Comment:
```suggestion
== WMCO supported installation method
```
If "Installation" should be capitalized here for some reason, feel free to ignore this suggestion.